performance-tracking/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/RuntimeContentProvider.java

```diff
@@ -80,7 +80,8 @@ public boolean onCreate() {
   private Config createConfig(
       @NonNull final Context context,
       @Nullable final ConfigurationResponse lastConfig,
-      @Nullable final String appId) {
+      @Nullable final String appId,
+      @NonNull final long maxMetricDuration) {
```

Comment:
Does it make sense for a `long` to have nullability?

Reply:
Oops, you're right, a `long` can't be null. I meant to remove the `@NonNull`.